doc/install_others.txt

```diff
@@ -25,18 +25,21 @@ focusing on how to install the appropriate dependencies.
 
 Nicolas Pinto provides `ebuild scripts <https://github.com/npinto/sekyfsr-gentoo-overlay/tree/master/sci-libs/Theano>`_.
 
-Docker images
--------------
+Docker
```

Comment:
Done

On Tue, Aug 30, 2016 at 10:28 AM, Frédéric Bastien <notifications@github.com
> wrote:

> In doc/install_others.txt
> <https://github.com/Theano/Theano/pull/4913#discussion_r76805810>:
>
> > @@ -25,18 +25,21 @@ focusing on how to install the appropriate dependencies.
> >
> >  Nicolas Pinto provides `ebuild scripts <https://github.com/npinto/sekyfsr-gentoo-overlay/tree/master/sci-libs/Theano>`_.
> >
> > -Docker images
> > --------------
> > +Docker
>
> I think porting this one to the 0.8.X branch would be good.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4913/files/da800d748bcd0c79d5bb1ad71b7eedb9a306afd6#r76805810>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARWYKertX7uJioetxflKfMZwr7TbrRScks5qlD4UgaJpZM4Jwjgo>
> .
>